at_response.c

```diff
@@ -1229,6 +1229,8 @@ static int at_response_cmgr (struct pvt* pvt, const char * str, size_t len)
 				return 0;
 			}
 
+			ast_log(LOG_NOTICE, "[%s] Incoming message parsed '%s'\n", PVT_ID(pvt), str);
+
 			ast_debug (1, "[%s] Successfully read SMS message\n", PVT_ID(pvt));
```

Comment:
I don't think this is worthy of a NOTICE message. You could've just appended the message-str to the ast_debug(1) which is already there.

```
[%s] Successfully read SMS message: %s\n
```

Sure you'll have to enable debug mode to get the debug logs, but that's also the case if you want to debug chan_sip protocol issues.